### DISCUSSION TITLE
Inconsistent overall status for jobs with continue-as-error.

### DISCUSSION THREAD

**Discussion by wsfulton** [community/community#13955] [Other Feature Feedback, Questions, & Ideas]
We use `continue-as-error` set to true in our ci.yml file so that we can mark some jobs as allowed to fail and not affect overall CI status:

https://github.com/swig/swig/blob/e4c972c328f7edd11570abeb6104775711e50db5/.github/workflows/ci.yml#L298-L301

which leads to inconsistencies in overall status. Sometimes the status shows green and sometimes red when the continue-on-error job does actually fail. 

Consider commit 0b157406 https://github.com/swig/swig/commit/0b157406638dc0a37b01a9f498577a2a72e9108c, which shows a red cross summary:

![Screenshot from 2022-03-31 08-13-43](https://user-images.githubusercontent.com/2878511/160998783-62e8769d-6b6c-44ea-adfc-860c545fdbd9.png)

The overall actions summary at https://github.com/swig/swig/actions/ shows a green tick:

![Screenshot from 2022-03-31 07-49-25](https://user-images.githubusercontent.com/2878511/160999554-bc21ec1c-cff2-41dd-b470-0c5f41a2260c.png)

The individual run https://github.com/swig/swig/actions/runs/2054493940 is a green tick:

![Screenshot from 2022-03-31 07-51-07](https://user-images.githubusercontent.com/2878511/161000755-d21016fa-df99-4525-bd1a-5333269d86a1.png)

And the git log https://github.com/swig/swig/commits/master shows a red cross:

![Screenshot from 2022-03-31 07-48-13](https://user-images.githubusercontent.com/2878511/161001051-90782e49-442d-43cf-8007-948b93bb70f8.png)

They should all be consistently green.

**Comment by muhammedsaidkaya** [comment]
I encounter with this problem. Is there any solution method?

[https://github.com/orgs/community/discussions/65359](https://github.com/orgs/community/discussions/65359)
